docs/extensions.md

```diff
@@ -55,6 +55,8 @@ non-existing metrics occur):
 * `3scale-limit-reset`: An integer stating the amount of seconds left for the
   current limiting period to elapse. A negative integer value means there is no
   limit in time.
+* `3scale-limit-max-value`: An integer stating the quota after the current
```

Comment:
`s/quota after the current limiting period ends/maximum number of hits allowed in this limiting period/`

Reply:
That's better :+1: 
Fixed.